build.gradle

```diff
@@ -52,9 +52,15 @@ sourceSets {
 
 repositories {
 	mavenCentral()
+	maven { url "https://repo1.maven.org/maven2/"}
```

Comment:
```suggestion
```
This is just the same as `mavenCentral()`, so not needed

Reply:
sorry, idk what am I doing.